api/api_public_blockchain.go

```diff
@@ -282,7 +282,7 @@ type CallArgs struct {
 	Input                hexutil.Bytes   `json:"input"`
 }
 
-func (args *CallArgs) data() []byte {
+func (args *CallArgs) Payload() []byte {
```

Comment:
```suggestion
func (args *CallArgs) InputData() []byte {
```
Instead of introducing a new name, we can use the existing names.

Reply:
Sure. Reflected at here: https://github.com/klaytn/klaytn/pull/1965/commits/39bb113b45c4b7e11445fae4f8b88d4a2ec1d5c5